src/pages/Scholarships.jsx

```diff
@@ -20,10 +20,11 @@ function ScholarshipsPage() {
   const [scholarships, setScholarships] = useState([]);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState();
+  const [sortList, setSortList] = useState('deadline');
```

Comment:
It's probably better to have two state values, `sort`/`sortField` and `sortDir`.

Reply:
Punted for #342